Sprint-2/debug/0.js

```diff
@@ -1,7 +1,22 @@
 // Predict and explain first...
 
 function multiply(a, b) {
-  console.log(a * b);
+  //console.log(a * b);
+  return a * b;
 }
 
 console.log(`The result of multiplying 10 and 32 is ${multiply(10, 32)}`);
+
+/* 
+Frankly speaking, I did this task in reverse order. 
```

Comment:
Yes, lots of people do this. We ask you to predict-explain because there's reasonable evidence it increases retention, understanding, and test scores. 

https://files.eric.ed.gov/fulltext/EJ1341654.pdf